application/forms/Config/UserBackend/LdapBackendForm.php

```diff
@@ -146,6 +146,7 @@ public function createElements(array $formData)
             'filter',
             array(
                 'preserveDefault'   => true,
+                'ignoreDefault'     => true,
```

Comment:
I'm afraid but this is not a static default what's inserted here, but a suggestion that's otherwise thrown away and never used if 'ignoreDefault' is set to true.